compiler/next/include/chpl/resolution/resolution-types.h

```diff
@@ -900,7 +900,94 @@ template<> struct update<owned<resolution::ResolvedFunction>> {
     return defaultUpdateOwned(keep, addin);
   }
 };
-/// \endcond
+
+template<> struct stringify<chpl::resolution::ResolvedExpression> {
+  std::string operator()(StringifyKind stringKind,
+                         const chpl::resolution::ResolvedExpression& stringMe) const {
+    return defaultStringify(stringKind, stringMe);
```

Comment:
Are all of these necessary? Won't the template choose defaultStringify if no other template works? Or these are just setting ourselves up for the future to implement?

Reply:
Right now the default template generates an error (it has `= 0`)